Waiter.py

```diff
@@ -0,0 +1,104 @@
+import os
+import re
+import time
+from colorama import Fore, Back, Style
+import pandas as pd
+from tabulate import tabulate as tb
+import numpy as np
+
+
+
+#Menu Options
+print("[ORDER & PAY HERE]")
+NumCust = input("How many of you will be dining with us today?")
+
+
+#Trying to figure out how to clear out the output to make output cleaner for user
+# def ClearOutput():
+#     if os.name == 'nt':
+#         os.system('cls')
+#     else:
+#         os.system('clear')
+# ClearOutput()
+
+#Item Details: Name & Prices ==> Just edit "ItemDet" to make future amendments
+ItemDet = {'Cheese Burger':6, 'Fries':3, 'Tenders': 4, 'Soda':2}
+OrderList = list(ItemDet.keys())
+ItemP = list(ItemDet.values())
+
+#Make list for KEYS
+numOrderList = int(len(OrderList))
+
+
+# print(Fore.RED+"[MENU]"
+#       "\nItem 1 [Cheese Burger]: $6"
+#       "\nItem 2 [Fries]: $3"
+#       "\nItem 3 [Tenders]: $4"
+#       "\nItem 4 [Soda]: $2")
+
+mntbl = {'Item #':list(range(1,numOrderList+1)),
+         'Item Name': OrderList,
+         'Item Price':ItemP}
+mntblDF = pd.DataFrame(mntbl, columns = ['Item #', 'Item Name', 'Item Price'])
+
+print(Fore.RED+"[MENU]")
+print(tb(mntblDF,headers='keys',tablefmt='psql'))
+
+
+#Make list for VALUES through user input
+OrderQuant = []
+for a in range(1,numOrderList+1):
```

Comment:
this makes the user iterate through every single item in the menu in the order. Make something that would allow the user to only state what they want to order.